README.md

```diff
@@ -4,26 +4,65 @@ This project provides information and is a place to store the results of the tra
 
 # DAY 2
 
-
 ## NX Workspace
+
 - What are the benefits of a monorepo?
+  - The main benefit of using a monorepo is code sharing and reusability.
+  - By storing all code for related projects or components in a single repository, developers can easily share and reuse code across different parts of the software.
+  - This promotes consistency, reduces duplication, and simplifies maintenance of shared components and libraries.
 - What is Nx workspace?
+  - Nx workspace is a development environment provided by the Nx toolset, designed for building and managing monorepos.
+  - Nx workspace provides a structured environment for developing multiple projects within a single repository.
 - Folder structure of Nx Workspace?
+  - myorg/  
+    ├── apps/  
+    ├── libs/  
+    ├── tools/  
+    ├── nx.json  
+    └── tsconfig.base.json
+  - `/apps/` contains the application projects. This is the main entry point for a runnable application.
+  - `/libs/` contains the library projects.
+  - `/tools/` contains scripts that act on your code base.
+  - `/nx.json` configures the Nx CLI itself. It tells Nx what needs to be cached, how to run tasks etc.
+  - `/tsconfig.base.json` sets up the global TypeScript settings and creates aliases for each library to aid when creating TS/JS imports.
 - How many Library Types of Nx Workspace?
```

Comment:
NX workspace defines 4 types of libraries:

* Feature libraries:
Developers should consider feature libraries as libraries that implement smart UI (with access to data sources) for specific business use cases or pages in an application.

* UI libraries:
A UI library contains only presentational components (also called "dumb" components).

* Data-access libraries:
A data-access library contains code for interacting with a back-end system. It also includes all the code related to state management.

* Utility libraries:
A utility library contains low-level utilities used by many libraries and applications.